tests/ui/coherence/coherence-impls-copy.stderr

```diff
@@ -59,18 +59,38 @@ LL | impl Copy for &'static mut MyType {}
    |               ^^^^^^^^^^^^^^^^^^^ type is not a structure or enumeration
 
 error[E0206]: the trait `Copy` cannot be implemented for this type
-  --> $DIR/coherence-impls-copy.rs:25:15
+  --> $DIR/coherence-impls-copy.rs:26:15
    |
 LL | impl Copy for (MyType, MyType) {}
    |               ^^^^^^^^^^^^^^^^ type is not a structure or enumeration
 
 error[E0206]: the trait `Copy` cannot be implemented for this type
-  --> $DIR/coherence-impls-copy.rs:30:15
+  --> $DIR/coherence-impls-copy.rs:31:15
    |
 LL | impl Copy for [MyType] {}
    |               ^^^^^^^^ type is not a structure or enumeration
 
-error: aborting due to 8 previous errors
+error[E0277]: the trait bound `&'static mut MyType: Clone` is not satisfied
+  --> $DIR/coherence-impls-copy.rs:21:15
+   |
+LL | impl Copy for &'static mut MyType {}
+   |               ^^^^^^^^^^^^^^^^^^^ the trait `Clone` is not implemented for `&'static mut MyType`
+   |
+   = help: the trait `Clone` is implemented for `MyType`
+note: required by a bound in `Copy`
+  --> $SRC_DIR/core/src/marker.rs:LL:COL
```

Comment:
These are unfortunate, almost tautological, I wonder if we could leverage some specialized obligation cause code to avoid them.

Reply:
resolved